components/integration/staging/base/kustomization.yaml

```diff
@@ -11,7 +11,16 @@ images:
   newName: quay.io/redhat-appstudio/integration-service
   newTag: f7bb0788791c37b7a82a98555dc710d434e36a56
 
+configMapGenerator:
+- name: console-url
+  literals:
+    - CONSOLE_URL=""
+
 namespace: integration-service
 
 patches:
   - path: manager_resources_patch.yaml
+  - path: config-console-url.yaml
```

Comment:
I don't understand why are we patching something that's defined in the same dir? Why not to define it directly?

Reply:
Removed, direct definition is sufficient enough for this case.